projects/Mallard/src/screens/article-screen.tsx

```diff
@@ -31,13 +32,18 @@ export interface ArticleTransitionProps {
 }
 
 const useArticleResponse = ({ collection, article, issue }: PathToArticle) => {
-    const resp = useJsonOrEndpoint<ArticleType>(
+    const resp = useJsonOrEndpoint<Collection>(
         issue,
-        `collection/${collection}`,
+        FSPaths.collection(issue, collection),
+        APIPaths.collection(collection),
     )
     if (resp.state === 'success') {
+        // TODO: we aren't storing the path anywhere on the article
+        // which means we can't key into our collection (which is keyed by path)
+        // even when we have an article
         const articleContent =
-            resp.response.articles && resp.response.articles[article]
+            resp.response.articles &&
+            Object.values(resp.response.articles).find(a => a.key === article)
```

Comment:
I'm confused about what is different here? `article.key` should match it's key on the collection

Reply:
I wondered about this too. At the moment it doesn't, `articles` is keyed by the CAPI path and `article.key` is the internal page code. @AWare could we change this?

Reply:
the backend shouldn't leak internal page codes as primary ids any more